src/components/cards/MyWallet/MyWallet.vue

```diff
@@ -0,0 +1,76 @@
+<template>
```

Comment:
Since this is a new component can we convert it to the new setup syntax?

Reply:
yeah I made a general comment on converting them once functionality was complete